client/Gruntfile.js

```diff
@@ -782,15 +782,55 @@ module.exports = function(grunt) {
 
   });
 
+
+  // Processes misc files included in the GlobaLeaks repository that need to be
+  // included as data like the license and changelog
+  grunt.registerTask('includeExternalFiles', function() {
+      var cnt = grunt.file.read('../LICENSE');
+      fs.writeFileSync('tmp/data/license.txt', cnt)
+
+      cnt = grunt.file.read('../CHANGELOG');
+      var lines = cnt.split("\n"),
+          chnglog = [],
+          obj;
+
+      for (var i = 0; i < lines.length; i++) {
+        var l = lines[i];
+        // Format of the first line of every new release is:
+        // Changes in version 2.76.4 - 2017-04-12 - alert:notice
```

Comment:
nice implementatio and ideas @NSkelsey 

but as discussed a week ago on slack also with @fpietrosanti let's avoid for now to implement filters.

prioritiziong means decide not to show something; i would suggest to skip until proper understanding of how to categorize messagges.